README.md

```diff
@@ -344,6 +344,72 @@ const Title = glamorous.h1({
 </ThemeProvider>
 ```
 
+`glamorous` also exports a `withTheme` higher order component (HOC) so you can access your theme in any component!
+> Try this out in your browser [here](https://codesandbox.io/s/qYmJjE4jy)!
+
+```jsx
+import glamorous, {ThemeProvider,  withTheme} from glamorous
+
+// our main theme object
+const theme = {
+  main: {color: 'red'}
+}
+
+// a themed <Title> component
+const Title = glamorous.h1({
+  fontSize: '10px'
+}, (props, theme) => ({
+  color: theme.main.color
+}))
+
+// normal component that takes a theme prop
+const SubTitle = ({children, theme: {color}}) => (
+  <h3 style={{color}}>{children}</h3>
+);
+
+// extended component with theme prop
+const ThemedSubTitle = withTheme(SubTitle);
+
+<ThemeProvider theme={theme}>
+  <Title>Hello!</Title>
+  <ThemedSubTitle>from withTheme!</ThemedSubTitle>
+</ThemeProvider>
+```
+
+Or if you prefer decorator syntax:
```

Comment:
Yeah I haven't looked back since I discovered that! :smile:
I have great one that is for checking internet connection `@connection` adds an offline prop that is a boolean. It's great.